### DISCUSSION TITLE
Long term data retention \ underlying HW

### DISCUSSION THREAD

**Discussion by kpnobvious** [jertel/vuegraf#90] [Q&A] [answered]
Hi Jertel,

Curious as i think you've been collecting emporia data for quite some time now what, if anything, you're doing with continuous queries and if you're trying to aggregate any data.

Also, what is your underlying HW you're using for docker?  I've started to run into high CPU\MEM utilization with a year old DB now.  Even though the DB is only about 1GB i think i'm hitting some limits on the 32bit RPI\influxdb 1.8.x and need to move to something beefier

**Comment by jertel** [comment, marked as answer]
My DB is about 5GB in size.  InfluxDB v1.8.4 averages less than 0.1% CPU (from a single core of a 10 core Intel Xeon 2.4GHz), briefly spiking to 15% during an update. It uses 350KB of resident memory, and 8MB of virtual memory. The only process that talks to InfluxDB is Vuegraf, and Grafana. I have about 8 alerts configured in Grafana.

I've not had any concerns with it on this setup. I haven't tried it on a Pi. Perhaps you could make a backup of the DB, then truncate it and see if the performance problems go away.
